src/test/java/com/segment/analytics/android/integrations/optimizelyx/OptimizelyXTest.java

```diff
@@ -192,7 +191,6 @@
   }
 
   @Test public void listenerDisabled() {
-    integration.listen = false;
     verify(client, times(0)).addNotificationListener(integration.listener);
```

Comment:
No need to change this now, but for the future it's clearer to do `never()` here.